policy/DecamMapper.paf

```diff
@@ -17,9 +17,9 @@ camera: "../decam/camGeom"
 
 exposures: {
     raw: {
-        template:    "%(visit)07d/decam%(visit)07d.fits[%(ccdnum)d]"
-        python:      "lsst.afw.image.ExposureF"
-        persistable: "ExposureF"
+        template:    "%(date)s/%(filter)s/decam%(visit)07d.fits.fz[%(hdu)d]"
+        python:      "lsst.afw.image.DecoratedImageF"
+        persistable: "DecoratedImageF"
```

Comment:
`DecoratedImage` may be deprecated, but it's the only thing we have that fits here.  The `Exposure` constructor will now produce a warning because they are not actually `Exposure`s (e.g., no separate `Wcs` extension), and plain `Image` doesn't give us the FITS header.